pkg/sql/importer/import_stmt_test.go

```diff
@@ -3151,26 +3151,40 @@ func TestImportIntoCSV(t *testing.T) {
 
 	// Verify a failed IMPORT INTO won't prevent a subsequent IMPORT INTO.
 	t.Run("import-into-checkpoint-leftover", func(t *testing.T) {
-		sqlDB.Exec(t, `CREATE TABLE t (a INT PRIMARY KEY, b STRING)`)
-		defer sqlDB.Exec(t, `DROP TABLE t`)
 
-		// Insert the test data
-		insert := []string{"''", "'text'", "'a'", "'e'", "'l'", "'t'", "'z'"}
+		for _, emptyTable := range []bool{true,false} {
+			subtestName := "empty-table"
+			if emptyTable == true {
+				subtestName = "nonEmptyTable"
+			}
+			t.Run(subtestName, func(t *testing.T){
```

Comment:
We should have another pair of subtests that enable/disable use of DeleteRange. Probably easier to implement if we also add that cluster setting.

Reply:
agreed. but will wait to implement until cluster setting question is answered.

Reply:
No cluster setting for now. May add during stability.